packages/core/src/http.listener.spec.ts

```diff
@@ -1,18 +1,15 @@
-import { map, mapTo, tap } from 'rxjs/operators';
+import { throwError } from 'rxjs';
+import { mapTo, switchMap } from 'rxjs/operators';
 import * as request from 'supertest';
-import { combineRoutes } from './effects/effects.combiner';
 import { Effect } from './effects/effects.interface';
-import { HttpRequest } from './http.interface';
 import { httpListener } from './http.listener';
-import { matchPath, use } from './operators';
+import { matchPath } from './operators';
+import { HttpError } from './util/error.util';
 
 describe('Http listener', () => {
-
   it('reacts to attached effect', async () => {
-    const effect$: Effect = req$ => req$.pipe(
-      matchPath('/test'),
-      mapTo({ status: 200 })
-    );
+    const effect$: Effect = req$ =>
+      req$.pipe(matchPath('/test'), mapTo({ status: 200 }));
```

Comment:
[suggestion] IMO more readable format is if `matchPath` and `mapTo` will be in new lines. I know that is `spec` file but the code should be always easy to read.

Reply:
yup, it is formatted by prettier - will change it